Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -7265,6 +7265,42 @@ template<CSSPropertyID property> RefPtr<CSSValue> consumeBackgroundSize(CSSParse
     return createPrimitiveValuePair(horizontal.releaseNonNull(), vertical.releaseNonNull(), identicalValueEncoding);
 }
 
+RefPtr<CSSValueList> consumeAlignTracks(CSSParserTokenRange& range)
+{    
+    RefPtr<CSSValueList> parsedValues = CSSValueList::createCommaSeparated();
+
+    do {
+        if (range.atEnd())
+            break;
+
+        auto value = consumeIdent<CSSValueNormal, CSSValueFlexStart, CSSValueFlexEnd, CSSValueCenter, CSSValueSpaceBetween, CSSValueSpaceAround, CSSValueSpaceEvenly, CSSValueStretch, CSSValueFirst, CSSValueLast, CSSValueBaseline, CSSValueUnsafe, CSSValueSafe, CSSValueStart, CSSValueEnd>(range);
+        if (value)
+            parsedValues->append(value.releaseNonNull());
+        else
+            return nullptr;
```

Comment:
This doesn't seem correct when I look at the MDN syntax. `align-tracks: safe` isn't valid, but this function will consider it valid.

Ditto for `align-tracks: first`. 

From what I'm seeing the syntax is identical to `align-content`, except for the comma separated bit. So you could call whatever `align-content` is using inside the loop.

Reply:
Just to clarify the align-content in CSSProperties.json.